api/proto/auth/client.go

```diff
@@ -0,0 +1,892 @@
+/*
```

Comment:
I think this file can just live under `api/`, why do you have so many sub packages here?